src/Radicle/Internal/PrimFns.hs

```diff
@@ -351,8 +352,8 @@ purePrimFns = fromList $ allDocs $
     , ( "/"
       , "Divides one number by another. Throws an exception if the second argument is 0."
       , twoArg "/" $ \case
-          (Number x, Number y) | y /= 0 -> pure $ Number (x / y)
-          (Number _, Number _) -> throwErrorHere $ OtherError "Can't divide by 0"
+          (Number _, Number 0) -> throwErrorHere $ OtherError "Can't divide by 0"
```

Comment:
I’m not sure this works in all cases. For example assume the rational type is implemented as an integer tuple and the literal `0` is converted to `(0, 1)`. It might be the case that we have a number that look like `(0, 2)`. This number is equal to `(0, 1)` but does not match `(0, 1)` because pattern matching works on representations.

Alltough the code might actually work it depends on the implementation of `Rational` so I’d rather err on the side of caution.